default/scripting/buildings/shipyards/ORBITAL_DRYDOCK.focs.txt

```diff
@@ -15,6 +15,24 @@ BuildingType
     ]
     EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
     effectsgroups = [
+        // With no pop, no repair or sitrep messages
+        EffectsGroup
+            scope = And [
+                Ship
+                InSystem id = Source.SystemID
+                OwnedBy empire = Source.Owner
+                Structure high = LocalCandidate.MaxStructure - 0.001
+            ]
+            activation = And [
+                Source
+                ContainedBy And [
+                    Planet
+                    Population high = 0
+                ]
+            ]
+            stackinggroup = "SHIP_REPAIR"
+            priority = 75
```

Comment:
That should work, thanks.

Still not ideal, because a change to the values of the priority macros could potentially mess up the interaction with the other repair techs/mechanics (which use those macros), but probably far better and more robust than the lonely "75" value we had before. At least changing the priority macros can't mess up drydock repair itself now.

Good enough for now. A more thorough solution is beyond the scope of his PR anyway, as that's a more general issue.